src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,31 @@
+import { useContext } from 'react';
+import { PeopleContext } from '../store/PeopleContent';
+import { PersonLink } from './PersonLink';
+
+export const PeopleTable: React.FC = () => {
+  const { people } = useContext(PeopleContext);
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map((personItem, index) => {
+          return <PersonLink key={index} person={personItem} />;
```

Comment:
Using the `index` as a key in the `map` function is not recommended because it can cause issues with component identity when the list changes. Consider using a unique identifier from the `personItem`, such as an `id` or `slug`, if available.